docs/source/gpu.rst

```diff
@@ -409,23 +409,36 @@ Geophysical subsurface characterization:
    when ``explicit=False``.
 
 
-Example
--------
+Examples
```

Comment:
I think we should move this section to be after "Overview" and have a separate section "Supported Operators" the full tables. Otherwise this section will be lost to the reader

Reply:
Good point :smile: